lib/smart_answer_flows/legalisation-document-checker/outcomes/outcome_results.govspeak.erb

```diff
@@ -21,6 +21,24 @@
       - [General Register Office for Northern Ireland](http://www.nidirect.gov.uk/general-register-office-for-northern-ireland)
 
       ^A photocopy of your document won’t be accepted.^
+    
+  <% when 'letter_no_trace' %>
+      ##<%= heading %>
+      Your <%= type %> can be legalised if it is either:
+
+      - an official certified copy of the letter signed by the General Register Office (GRO) with the original seal
+      - an original letter (or original certified copy) issued by a local register office, with the original signature of the official who issued the letter
+
+      It can take longer to legalise a signature on an original letter, especially if it was issued before 1991.
+
+      It might be quicker to order a certified copy of your letter from the:
+
+      - [GRO (England and Wales)](https://www.gro.gov.uk/gro/content/certificates/default.asp)
+      - [GRO for Scotland](http://www.gro-scotland.gov.uk/contacts/opentime/index.html/)
+      - [General Register Office for Northern Ireland](http://www.nidirect.gov.uk/general-register-office-for-northern-ireland)
+
+      ^A photocopy of your document won’t be accepted.^  
+    
```

Comment:
@ikennaokpala: Minor: Given that this is mostly a copy of the `birth_death` section above, it might be worth considering ways to reduce the amount of duplication.